x-pack/plugins/security_solution/cypress/e2e/detection_rules/custom_query_rule.cy.ts

```diff
@@ -105,98 +133,122 @@ describe('Custom query rules', () => {
     login();
   });
   describe('Custom detection rules creation', () => {
-    const expectedUrls = getNewRule().referenceUrls?.join('');
-    const expectedFalsePositives = getNewRule().falsePositivesExamples?.join('');
-    const expectedTags = getNewRule().tags?.join('');
-    const mitreAttack = getNewRule().mitre as Mitre[];
-    const expectedMitre = formatMitreAttackDescription(mitreAttack);
     const expectedNumberOfRules = 1;
 
     beforeEach(() => {
-      const timeline = getNewRule().timeline as CompleteTimeline;
       deleteAlertsAndRules();
-      createTimeline(timeline).then((response) => {
-        cy.wrap({
-          ...getNewRule(),
-          timeline: {
-            ...timeline,
-            id: response.body.data.persistTimeline.timeline.savedObjectId,
-          },
-        }).as('rule');
-      });
+      createTimeline(getTimeline())
+        .then((response) => {
+          return response.body.data.persistTimeline.timeline.savedObjectId;
+        })
+        .as('timelineId');
     });
 
     it('Creates and enables a new rule', function () {
       visit(RULE_CREATION);
-      fillDefineCustomRuleAndContinue(this.rule);
-      fillAboutRuleAndContinue(this.rule);
-      fillScheduleRuleAndContinue(this.rule);
+
+      cy.log('Filling define section');
+      importSavedQuery(this.timelineId);
+      continueWithNextSection();
+
+      cy.log('Filling about section');
+      fillRuleName();
+      fillDescription();
+      fillSeverity();
+      fillRiskScore();
+      fillRuleTags();
+      expandAdvancedSettings();
+      fillReferenceUrls();
+      fillFalsePositiveExamples();
+      fillThreat();
+      fillThreatTechnique();
+      fillThreatSubtechnique();
+      fillNote();
+      continueWithNextSection();
```

Comment:
I'd group these into an object as well which would improve readability, encapsulate details and simplify imports:

```ts
aboutStep.fillRuleName();
aboutStep.fillDescription();
// etc
```